### ISSUE TITLE
Subtitles

### ISSUE THREAD

**Issue by traidor** [subhra74/xdm#597]
## Checklist

- [ ] I'm reporting a feature request
- [ ] I've verified that I'm running xdm latest version
- [ ] I've searched the bugtracker for similar feature requests including closed ones

## Description

WRITE DESCRIPTION HERE

**Comment by traidor** [comment]
Would be great a subtitle integration in download or a subtitle plug-in.

**Comment by dp876** [comment]
that's redundant... if u watch movies with vlc... click 'view' in the toolbar, then click 'VLsub'....

**Comment by CrimsonBloodfang** [comment]
Yes, wish it could grab subtitles as well from the web player. There are not so popular shows that have no subtitles in opensubtitles or not found anywhere other than the website's own translation/subtitle. 
I really don't want to have to use 1DM+ for Android in a Waydroid container just for this functionality.

**Comment by fcakyon** [comment]
@subhra74, thanks for the fantastic tool! Are there any plans for the subtitles download feature?
